release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -24,7 +24,7 @@ Red Hat {product-title}
 (link:https://access.redhat.com/errata/RHBA-2020:0062[RHBA-2020:0062]) is now
 available. This release uses Kubernetes 1.16 with CRI-O runtime. This is an
 increase of two versions of Kubernetes from {product-title} 4.2, which used
-Kubernetes 1.14. New features, changes, and known issues that pertain to
+Kubernetes 1.14. See the link:https://v1-15.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.15 release notes] and link:https://v1-16.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.16 release notes] for details. New features, changes, and known issues that pertain to
```

Comment:
@bergerhoffer i wonder if we can find some prominent place for this link and also highlight the fact this release notes is carrying the Kubernetes API deprecation notices.